### DISCUSSION TITLE
feature request: add torrent completion percent number

### DISCUSSION THREAD

**Discussion by 4nd3r** [jesec/flood#660] [Ideas]
Usecase: I need to recheck torrent hash and after it completes, it's not completely obvious (at least for me) if it's 99,9% or 100%. So I always open torrent details and verify the percent number. This change also helps to notice "stuck torrents" more conveniently.

Quick dirty example:

![image](https://github.com/jesec/flood/assets/7166727/339ddb85-0609-4ab2-adc2-0aed2039cc0e)

**Comment by KlutzyBubbles** [comment]
Also looking for something like this, it is especially difficult to tell whether a paused/stopped torrent is complete or very close to complete without resuming it to see if it goes green or blue.

This image is an example of it happening, the stopped torrent in the 2nd row is actually 99.8% complete, but indistinguishable from the other stopped torrent which is 100% complete. 

![image](https://github.com/user-attachments/assets/2c5ed638-786a-4237-9f45-ef9a92137698)
